src/pages/admin/student/edit/actions/summaryStudentRequest.ts

```diff
@@ -0,0 +1,24 @@
+import { adminActionEnums } from '../../../../../common/actionEnums/admin';
+import { StudentSummary } from '../../../../../model/studentSummary';
+import { studentApi } from '../../../../../rest-api';
+
+export const summaryStudentByIdRequestStarted  = (studentId: number) => {
+  return function(dispatcher) {
+    const promise = studentApi.getStudentById(studentId);
+
+    promise.then(
+      (data) => {
+        dispatcher(summaryStudentByIdRequestCompleted(data));
+      },
+    );
+
+    return promise;
+  };
+};
+
+export const summaryStudentByIdRequestCompleted  = (student: StudentSummary) => {
+  return {
+      payload: student,
```

Comment:
It looks like there are 4 spaces indentation instead of 2, isn't it?

All files should have 2 spaces